filter/etc/pam.d/rspamd

```diff
@@ -0,0 +1,5 @@
+auth [success=1 default=ignore] pam_succeed_if.so user in admin
+auth [success=ok default=die] pam_succeed_if.so user ingroup [domain admins]
```

Comment:
Did you find any reason to remove the esmith template for this file? `admin` and `domain admins` are DB values: I remember this discussion

https://community.nethserver.org/t/need-to-share-on-the-defunct-admin-user/4630

And manual page

http://docs.nethserver.org/en/v7/accounts.html#admin-account

We probably need to document the `admins` key meaning in a better way here: https://github.com/NethServer/docs/blob/master/developer-manual/en/nethserver-sssd.rst